pkg/asset/machines/worker.go

```diff
@@ -223,6 +223,14 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 			}
 			machineConfigs = append(machineConfigs, ignFIPS)
 		}
+		if len(ic.Workload) > 0 {
+			w := ic.Workload[0]
+			ignWorkload, err := machineconfig.ForWorkloadPartitioning(string(w.Name), w.CPUIds, "worker")
```

Comment:
This gives me pause. I've asked a question in the enhancement PR about whether the same cpuset should be used for master and workers. See https://github.com/openshift/enhancements/pull/703#pullrequestreview-624640278.

Reply:
Given that this will only be used for SNO in 4.8 it probably doesn't matter, but I agree it's worth designing this right out of the gate.

Reply:
For this first release it probably doesn't matter, and if we want different pools in the future, perhaps it's not too hard to eventually also allow the same "WorkloadSettings" in install-config.yaml to be added under the `compute` and `controlPlane` in addition to it being globally available at the top level?